pkgs/applications/office/beamerpresenter/default.nix

```diff
@@ -1,39 +1,36 @@
 { lib, mkDerivation, fetchFromGitHub, installShellFiles,
-  qmake, qtbase, poppler, qtmultimedia }:
+  qmake, qtbase, qtmultimedia, mupdf, jbig2dec, openjpeg, gumbo }:
 
 mkDerivation rec {
   pname = "beamerpresenter";
-  version = "0.1.3";
+  version = "0.2.0";
 
   src = fetchFromGitHub {
     owner = "stiglers-eponym";
     repo = "BeamerPresenter";
     rev = "v${version}";
-    sha256 = "1nbcqrfdjcsc6czqk1v163whka4x1w883b1298aws8yi7vac4f1i";
+    sha256 = "10i5nc5b5syaqvsixam4lmfiz3b5cphbjfgfqavi5jilq769792a";
   };
 
   nativeBuildInputs = [ qmake installShellFiles ];
-  buildInputs = [ qtbase qtmultimedia poppler ];
+  buildInputs = [ qtbase qtmultimedia mupdf jbig2dec openjpeg gumbo ];
+
+  qmakeFlags = [ "RENDERER=mupdf" ];
```

Comment:
Is mupdf preferred upstream or should we make this configurable?

Reply:
The [AUR package] made by the author of the software seems to have switched to mupdf.
The README says that it can be built with both at the same time but this currently results in a [segfault].

[AUR package]: https://aur.archlinux.org/packages/beamerpresenter/
[segfault]: https://github.com/stiglers-eponym/BeamerPresenter/tree/v0.2.0#known-problems

Reply:
My bad, the [AUR package actually still supports both mupdf and poppler][PKGBUILD].
How can I make the package support building with mupdf, poppler, or both?

[PKGBUILD]: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=beamerpresenter&id=20cd2c5ff0f17230dba26edcccd033c52f2ccbb9#n24

Reply:
> How can I make the package support building with mupdf, poppler, or both?

Add a `renderer ? "mupdf"` option and use `{ inherit mupdf poppler; }.${renderer}` for `buildInputs`?

Reply:
It's amended.